ssh/ssher.py

```diff
@@ -91,6 +91,7 @@ def open_tunnel(user: str, key: str, host: str, port: int=22) -> Tunnelled:
             '-oControlPath=' + temp_paths[0],
             '-oStrictHostKeyChecking=no',
             '-oUserKnownHostsFile=/dev/null',
+            '-oLogLevel=ERROR',
```

Comment:
This is just one additional log level above the default, yeah? (Default being INFO.) The fact that it says "Warning" in front of the message doesn't mean we'll be missing messages at the WARNING level, because no such level exists. (I'm going off one single man page review, but wanted to make sure this doesn't hide any actual warning messages that we may care about in the future....)